webapp/migrations/Version20250309122806.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250309122806 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add sort keys to rankcache, allowing us to support different scoring functions efficiently and elegantly';
```

Comment:
I'd consider going a step further and not use these opaque sort keys directly to determine an ordering, but maybe a fuction of that data for flexibility. But then again, that might be not to hard to extend to later.

Reply:
I responded in the main conversation thread to this as well - we can always add the function if we need it later (I suspect we don't).